include/aspect/compat.h

```diff
@@ -65,6 +65,347 @@ namespace aspect
       }
   };
 }
+
+#include <deal.II/grid/tria_boundary_lib.h>
+
+
+namespace dealii
```

Comment:
bad idea -- this namespace belongs to deal.II :-) Only put things into the `aspect` namespace.